.gitignore

```diff
@@ -5,9 +5,17 @@ venv
 .vscode
 .env
 
-*.geojson
+# ignore files in scratch subfolder in plots and data_output folders
+!data_output/scratch/*
```

Comment:
A bit confused by this -- do we want both `!path` and `path`? I thought `!` would cancel this out

Reply:
Yeah this is confusing. I probably should have added a comment for this. So `!` acts as a keep because it is a `.gitignore` file. `!data_output/scratch/*` says to keep the folder structure and `data_output/scratch/*` says to ignore the contents of the subfolder `scratch`. This way, the folders will exist, but files added to `scratch` won't be under version control. See [https://stackoverflow.com/questions/1248570/how-do-i-tell-git-to-ignore-everything-except-a-subdirectory](https://stackoverflow.com/questions/1248570/how-do-i-tell-git-to-ignore-everything-except-a-subdirectory).